simd/arm64/jsimd_neon.S

```diff
@@ -31,6 +31,254 @@
 .section .note.GNU-stack, "", %progbits  /* mark stack as non-executable */
 #endif
 
+.rodata
```

Comment:
Just a guess, should this be:

```assembly
-.rodata
+.section rodata
```
?

Reply:
Yup I think you're right, this section needs to be defined differently. I'll push a fix in the morning. Thanks!